GEOS-ESM/mepo
# Make mepo a pip-installable Python project

1. Renamed

- `mepo.d` -> `src/mepo`
- `src/mepo/utest` -> `tests`
- `doc` -> `docs`

3. Added `pyproject.toml` for pip installation

4. Helper script `mepo` moved to `bin/`


TODO:
- Check version number, in `pyproject.toml`
- Create account on PyPI
